hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/staking/StakingRewardsHandlerImplTest.java

```diff
@@ -94,6 +98,28 @@ public void setUp() {
         subject = new StakingRewardsHandlerImpl(rewardsPayer, stakePeriodManager, stakeInfoHelper);
     }
 
+    @Test
+    void testCoverageForPrivateConstructor()
+            throws NoSuchMethodException, InstantiationException, IllegalAccessException {
+        final Constructor<StakingUtilities> constructor = StakingUtilities.class.getDeclaredConstructor();
+        constructor.setAccessible(true);
+        try {
+            constructor.newInstance();
+        } catch (InvocationTargetException e) {
+            Throwable cause = e.getCause();
+            assertEquals(UnsupportedOperationException.class, cause.getClass());
+        }
+    }
+
+    @Test
+    void testStakeMetaChangesForNullOriginalAccount() {
+        noStakeChanges();
+        final var rewards = subject.applyStakingRewards(context, Collections.emptySet(), emptyMap());
+        final var modifiedAccount = writableAccountStore.get(payerId);
+        assert modifiedAccount != null;
```

Comment:
@thomas-swirlds-labs As I know, we decided to use assertJ that's why  `assertThat(modifiedAccount).isNotNull()` will be more aligned with assertJ. other than that all look good

Reply:
+1

Reply:
Updated. Thanks @povolev15